src/index.ts

```diff
@@ -0,0 +1,131 @@
+import express, { Express, NextFunction, Request, Response, Router, response } from 'express';
+import dotenv from 'dotenv';
+import { ContactRepository } from './repositories/contactRepository';
+import { DbUtils } from '../utils/dbutils';
+const cors = require('cors');
```

Comment:
```suggestion
import cors from 'cors';
```

+ `npm i --dev @types/cors`